apps/web/src/ee/clerk/components/OrganizationSwitcher.tsx

```diff
@@ -0,0 +1,21 @@
+import { OrganizationSwitcher as ClerkOrganizationSwitcher } from '@clerk/clerk-react';
+
+// TODO: this is tmp. styling
+const OrganizationSwitcherAppearance = {
+  elements: {
+    organizationSwitcherTrigger: {
+      paddingInline: 'var(--nv-spacing-125)',
+      fontSize: '14px',
+      width: '240px',
+      height: '52px',
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please use tokens for the rest of these, especially font size‏

Reply:
I've changed the spacing to use tokens and removed the fontSize -- This is just temporary so it roughly fits and we can start testing it - we will open new PR with custom styling by Nikolay for all the Clerk components.